app/assets/stylesheets/posts.scss

```diff
@@ -0,0 +1,26 @@
+// Place all the styles related to the posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+.post-comment {
+  height: 200px;
+}
+
+.post-comment__textarea {
+  height: 80% !important;
```

Comment:
suggested that setting height in actual px instead of %, since simple form seems to generate a div to wrap the fields with errors after validation. Since the height of its parent has changed, the height of the input box will change as well